python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -225,13 +226,18 @@ def edge_betweenness_centrality(
         raise TypeError("result type can only be np.float32 or np.float64")
 
     df = edge_betweenness_centrality_wrapper.edge_betweenness_centrality(
-        G, normalized, weight, k, vertices, result_dtype
+        G, normalized, weight, vertices, result_dtype
     )
 
     if G.renumbered:
         df = G.unrenumber(df, "src")
         df = G.unrenumber(df, "dst")
 
+    if type(G) is cugraph.Graph:
+        lower_triangle = df['src'] >= df['dst']
+        df[["src", "dst"]][lower_triangle] = df[["dst", "src"]][lower_triangle]
```

Comment:
Does this return the lower triangle? or do we want this to be df = df[["dst", "src"]][lower_triangle]

Reply:
This swaps `dst` and `src` values when `src` >= `dst` so that we have the same number of elements in the DataFrame but we ensure that all edges are respecting `src` < `dst` before merging them together, we do not  want to get rid of any values as we will sum scores among similar edges (i.e (src, dst, score): (0, 1, 0.5) and (1, 0, 0.25) should be viewed as (0, 1, 0.5) and (0, 1, 0.25) so that we can merge the scores and obtain (0, 1, 0.75))